src/deploy/functions/prepare.ts

```diff
@@ -37,8 +37,7 @@ import { applyBackendHashToBackends } from "./cache/applyHash";
 import { allEndpoints, Backend } from "./backend";
 import { assertExhaustive } from "../../functional";
 import { hostingConfig } from "../../hosting/config";
-import { getExistingRunRewrites } from "../hosting/convertConfig";
-import { ensureTargeted } from "../../functions/ensureTargeted";
+import { getExistingRewrites } from "../hosting/convertConfig";
```

Comment:
I specifically think we should try to solve this problem in just deploy/hosting/prepare.ts and nowhere else. I don't understand any need to put this in functions code.

Reply:
Agreed, now that I don't need to detect the codebase, just haven't moved it yet